libsol/transaction_printers.c

```diff
@@ -630,8 +663,19 @@ int print_transaction(const PrintConfig* print_config,
         const InstructionInfo* nonce_info = infos[0];
         print_system_nonced_transaction_sentinel(&(nonce_info->system), print_config);
         // offset parameters given to print_transaction_nonce_processed()
-        infos++;
-        infos_length--;
+        ADVANCE_INFOS()
+    }
+
+    if (infos_length > 1) {
```

Comment:
i think this check is unnecessary. the one above for durable nonce transactions is to discern between a durable nonce transaction (an otherwise valid instructions vector, with an AdvanceNonce instruction prefix) and a single instruction transaction that passes an AdvanceNonce instruction.  there is no analog case for ComputeBudget where it would make sense send a transaction with only `ComputeBudget` instructions